src/components/DVDsItemsandlists/DVDsItem.css

```diff
@@ -0,0 +1,64 @@
+  ul {
+    box-sizing: border-box;
+    list-style-type: none;
+    display: flex;
+    flex-wrap : wrap;
+    gap: 5px;
+    justify-content: space-around;
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
```

Comment:
you probably shouldn't have changed this, i think there was a mixup about the genre vs the whole film

Reply:
I have no idea how to fix this. When I change it to a class name it doesn't work.

Reply:
Fixed it, the ul was being used in a different file without being imported directly.